classes/metric/base.php

```diff
@@ -125,6 +125,29 @@ public function get_settings_url() {
         return null;
     }
 
+    /**
+     * The range for which a backfilled metric has retrieved data.
+     *
+     * @return string|bool
+     * @throws \dml_exception
+     */
+    public function get_range_retrieved() {
```

Comment:
would it be nicer to the calling code if this returned an array of two values? would remove a lot of explodes around the place

Reply:
Fixed